service/ztests/curl-pool-get.yaml

```diff
@@ -1,12 +1,15 @@
+skip: GET /pool/* is deprecated
```

Comment:
Revert the changes here and keep this test until we kill `GET /pool` and `GET /pool/{pool}`?